dashboard/test/models/unit_group_test.rb

```diff
@@ -229,17 +229,12 @@ class NameValidationTests < ActiveSupport::TestCase
     assert unit_group.stable?
   end
 
-  test "stable?: true if unit_group is not in a family" do
```

Comment:
I removed this test because I removed this special case from the stable? method. I went and looked to figure out what courses are not plc_courses and don't have a family name. The list was:

allthethingscourse 
andrea-test-dlp
fit2019-apprentice 
time4cs-control 
time4cs-experiment
mike-test 
self-paced-pl-csd-2021
self-paced-pl-csp-2021 

These all have the published_state of beta and I think that's the correct state for them to be in. If something is off about this we can always just update the published_state to stable.

Reply:
I want to look at some of the courses without family names to make sure they are still showing up in the dropdown. Just leaving here incase I don't get to it. Examples here I think include many of the hour of code scripts (flappy, starwars, etc)

Reply:
I realized I was talking about scripts without unit_groups here so I actually think this is not an issue but will check anyway

Reply:
All good. List looks right.